frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -869,10 +861,6 @@ export const experimentLogic = kea<experimentLogicType>([
     })),
     urlToAction(({ actions, values }) => ({
         '/experiments/:id': ({ id }, _, __, currentLocation, previousLocation) => {
-            if (!values.hasAvailableFeature(AvailableFeature.EXPERIMENTATION)) {
-                router.actions.push('/experiments')
-                return
-            }
```

Comment:
This was causing a bug (at least in Storybook, though I haven't seen it in prod), where opening the experiment view only worked some of the time – often available features didn't manage to load in time before this listener being called, so this logic initiated a redirect to the experiments list.
Replaced redirection with simply showing the experiments pay gate in `Experiment` same as in `Experiments`.